### ISSUE TITLE
MakeSrcFiles conversion should handle script in a different location

### ISSUE THREAD

**Issue by ghost** [KeyWorksRW/ttBld#30]
### Description:
Script files like .vcxproj don't need to be in the same directory as the source files. Even if they are, there's no requirement for .srcfiles to be in that directory. So, when we are converting a script we need to first convert the source file relative to the script, and then convert it to be relative to the location of .srcfiles.

```
foo/
    .srcfiles
    src/
        foo.cpp
        build/
            foo,vcxproj
```

foo.vcxproj will refer to `../foo.cpp`, but for it to work in .srcfiles, it needs to be `src/foo.cpp`.

Bottom line is that when converting, we need to keep in mind that the original script, the source files, and the .srcfiles we are creating can all be in different directories. We must be certain that they ultimately end up relative to the location of .srcfiles for our own **MakeNinja** to generate valid build scripts.

### Issue Type:
- [x] Bug report
- [ ] Feature request
- [ ] Other
